cryptography/src/bls12381/scheme.rs

```diff
@@ -1,4 +1,4 @@
-//! BLS12-381 implementation of the `Scheme` trait.
+//! BLS12-381 implementation of the [crate::Signer] trait.
```

Comment:
The rationale for `Scheme` here is that it includes both the signer and verifier. Do you see this differently?

Reply:
>pub trait Scheme: Signer + Verifier {}

impl<S: Signer + Verifier> Scheme for S {}

If this doesn't exist, that was an oversight. We should definitely have that.

Reply:
🤦

Reply:
Brain had clearly left the building when reviewing this.